app/assets/javascripts/components/physical_infrastructures/physical-server-toolbar.js

```diff
@@ -0,0 +1,50 @@
+ManageIQ.angular.app.component('physicalServerToolbar', {
+  bindings: {
+    physicalServerId: '=?',
+  },
+  controllerAs: 'vm',
```

Comment:
True. Although it just seems a bit odd to call `vm` with a different name. Probably because I have never come across any code that uses a different naming convention.

Anyway, per your request, I will be changing it to `toolbar`
We can always change it back to `vm` if requried ;-)